dcc-portal-ui/app/scripts/common/js/location.js

```diff
@@ -141,6 +141,11 @@
       getPaginationParams: function(dataType){
         return _.defaults({}, this.getJsonParam(dataType), {from: 1, size: 10});
       },
+      goToFirstPageOfDataType: function(dataType) {
+        var s = $location.search();
+        s[dataType] = JSON.stringify(_.extend(JSON.parse(s[dataType] || '{}'), {from: 1}));
```

Comment:
🇹🇭 
didn't read through getJsonParam to see what it did before
will refactor to use getJsonParam and setJsonParam instead